xml/System.Runtime.InteropServices/Marshal.xml

```diff
@@ -7632,7 +7632,7 @@ On .NET 6 and later versions, this method is functionally equivalent to <xref:Sy
 
 ## Remarks
  <xref:System.Runtime.InteropServices.Marshal.PtrToStructure%60%601%28System.IntPtr%29> is often necessary in COM interop and platform invoke when structure parameters are represented as <xref:System.IntPtr?displayProperty=nameWithType> values. You can pass a value type to this method overload.
- If the `ptr` parameter equals <xref:System.IntPtr.Zero?displayProperty=nameWithType>, `null` will be returned.
+ If the `ptr` parameter equals <xref:System.IntPtr.Zero?displayProperty=nameWithType> and `T` is not a value type, `null` is returned.  If `ptr` equals <xref:System.IntPtr.Zero?displayProperty=nameWithType> and `T` *is* a value type, a <xref:System.NullReferenceException> is thrown.
```

Comment:
Is `reference type` a better term than `not a value type`?

Reply:
I struggled with that very question. I defer to those who are closer to API design. If you prefer that I am fine changing it.

Reply:
`reference type` vs `value type` is typically how I think of it. But that might not fit with what's best for the docs or other users, so I'd defer to @gewarren and others.

Reply:
I'm fine with `reference type`. But then I wouldn't italicize `is` in the second sentence.

Reply:
fwiw I would expect 'reference type'. Otherwise I was wondering -- what is this 3rd kind of type that you are carefully including?

Reply:
Thanks all. Appreciate the discussion. I've updated this section and copied remarks to another overload I missed.